Content.Client/StationRecords/GeneralStationRecordConsoleWindow.xaml.cs

```diff
@@ -91,30 +94,50 @@ private void PopulateRecordContainer(GeneralStationRecord record)
     {
         RecordContainer.DisposeAllChildren();
         RecordContainer.RemoveAllChildren();
+
+
+        var name = new Label()
```

Comment:
you don't have to take this out of the recordControls array, as long as you put the controls you want in that container in the correct order it will appear in the correct order (the alternative is to just repeat `RecordContainer.AddChild(control)` everywhere - please don't do a hybrid of both)